ansible/playbooks/roles/opensearch/defaults/main.yml

```diff
@@ -3,16 +3,16 @@
 
 versions:
   RedHat:
-    elasticsearch_oss: "7.10.2"
-    opendistro: "1.13.*"
+    ops_version: "1.2.4"
+    ops_perftop_version: "1.1.0.0"
   Debian:
-    elasticsearch_oss: "7.10.2"
-    opendistro: "1.13.*"
+    ops_version: "1.2.4"
+    ops_perftop_version: "1.1.0.0"
 certificates:
   dirs:
-    certs:  /etc/elasticsearch
-    ca_key: /etc/elasticsearch/private
-    csr:    /etc/elasticsearch/csr
+    certs:  /usr/share/opensearch/config
+    ca_key: /usr/share/opensearch/config
```

Comment:
Previously we had separate paths to limit access to private keys (`mode: u=rwx,g=rx,o=  # elasticsearch.service requires 'rx' for group`) without limiting access to public certs. Is it possible to keep this approach?

Reply:
I've ran many tests to check configuration proposed by You which LGTM but I was not able to succeed.
Probably due to the fact the documentation is suggesting that these files need to be placed in relative path udder the `config` dir  ([[1](https://opensearch.org/docs/latest/upgrade-to/upgrade-to/#upgrade-to-opensearch)],[[2](https://docs.search-guard.com/latest/configuring-tls#using-x509-pem-certificates-and-pkcs-8-keys)]).This is why I put it there.
 However we can still open a spike to investigate it later on.